docs/source/components.rst

```diff
@@ -240,6 +261,12 @@ Cross field validation is made possible by overriding ``Form.clean()``:
             return data
 
 
+If you use a form's cleaned data to create a new model then model validation
+will also take place. It can be customized by passing a list of validators to a
```

Comment:
Yes, but only `ModelForm` automatically calls model validation. The Model itself does **not** automatically validate. `Model.full_clean` is not called in [`Queryset.create`](https://github.com/django/django/blob/15a20dc9aff1bd8a055ee7c322631b3ca4d1c474/django/db/models/query.py#L395-L403) or [`Model.__init__`](https://github.com/django/django/blob/15a20dc9aff1bd8a055ee7c322631b3ca4d1c474/django/db/models/base.py#L358-L453).